### ISSUE TITLE
PHP heredocs with double quotes stop doxygen

### ISSUE THREAD

**Issue by max1josef** [doxygen/doxygen#11107]
**Describe the bug**
PHP heredocs in the form `$msg = <<<"EOD"` (i.e. with double quotes) will stop doxygen from processing the remainder of the source file.

**To Reproduce**
```
<?php

/**
 * function with "heredoc"
 */

function hello()
{
    $msg = <<<"EOD"
Hello, World!
EOD;
    print($msg);
}

/**
 * this function is not documented by doxygen
 */

function foo()
{
    print("foo");
}

hello();
foo();
```

**Expected behavior**
Both functions should appear in the resulting documentation, but function `foo()` is omitted.
If you write:

* `    $msg = <<<EOD // herdoc without double quotes`
or
* `    $msg = <<<'EOD' // nowdoc`

doxygen works as expected.

**Version**
```
C:\>doxygen --version
1.12.0 (c73f5d30f9e8b1df5ba15a1d064ff2067cbb8267)
```
Platform is Windows 10, 64bit, 22H2, Build 19045.4780

**Additional context**
Not a big deal, since `$msg = <<<EOD` and `$msg = <<<"EOD"` are equivalent (i.e. the double quotes are optional). But I prefer the syntax with double quotes, to explicitely distingiush heredocs from nowdocs (with single quotes).

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11108 (including extended example)

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
